value.go

```diff
@@ -1052,8 +1053,8 @@ func discardEntry(e Entry, vs y.ValueStruct, db *DB) bool {
 }
 
 func (vlog *valueLog) doRunGC(lf *logFile) error {
-	_, span := otrace.StartSpan(context.Background(), "Badger.GC")
-	span.Annotatef(nil, "GC rewrite for: %v", lf.path)
+	_, span := otel.Tracer("Badger").Start(context.TODO(), "GC")
```

Comment:
same here, otel.Tracer("") so that the default one gets picked up
and "GC" --> "Badger.GC"